config/eol.yaml

```diff
@@ -0,0 +1,32 @@
+# Service EOLs (YYYY-MM-DD).
+php:
+  - version: '7.1'
+    eol: 2019-12-31
+  - version: '7.2'
+    eol: 2020-12-31
+  - version: '7.3'
+    eol: 2021-12-31
+mysql:
```

Comment:
Please re-name to mariadb, since we don't use MySQL.
Also version 10.0 is missing
We also don't have 10.3 I think